src/TerminalWidget.vala

```diff
@@ -53,6 +52,18 @@ namespace PantheonTerminal {
         public Granite.Widgets.Tab tab;
         public string? uri;
 
+        private string _tab_name = "";
```

Comment:
When duplicates were being distinguished by numbers, the tab label was different from the tab name; but now I agree there is no need to distinguish.